### DISCUSSION TITLE
automatic name assignement at discovery

### DISCUSSION THREAD

**Discussion by jujumo** [zwave-js/zwave-js#3850] [Ideas]
At discovery, devices are assigned a name.
In my case of name collision, suffix number is appended (e.g. cover.roller_shutter_3_2_3).
But for some reasons, the suffix is also happened in other cases (secondary service)
It make it very difficult to bulk rename them.
Especially when there is a lot of similar devices.

Is possible to include trailing part of the MAC address to ensure uniqueness of name, instead of appending this number for duplicates ?

**Comment by AlCalzone** [comment]
I think you meant to open this in https://github.com/home-assistant/core
